util/markdownlint.rb

```diff
@@ -24,3 +24,4 @@
 exclude_rule 'MD026' # Trailing punctuation in header
 exclude_rule 'MD029' # Ordered list item prefix
 exclude_rule 'MD030' # Spaces after list markers (default: 1!)
+exclude_rule 'MD040' # Allow fenced code blocks without a language
```

Comment:
Is this really required?  I'd prefer to see all blocks annotated even if it is just:

```text
using ```text for non-highlighted blocks
```

Reply:
I could live with that, but I think it would require an extensive pass to get it right.